selfdrive/car/chrysler/values.py

```diff
@@ -177,6 +187,47 @@ class ChryslerCarInfo(CarInfo):
       b'68500483AB',
     ],
   },
+
+  CAR.RAM_HD: {
+    (Ecu.combinationMeter, 0x742, None): [
+      b'68361606AH',
+      b'68492693AD',
+      b'68525485AB', #TODO: CHECK in newest updat on dongle 5c05760b592aee2c
```

Comment:
should I remove this?

Reply:
I do not feel it is needed for fingerprinting. So, it can be removed.

Reply:
I also, do not think gateway is needed on any of the chryslers